app/controllers/PayeControllerHistoric.scala

```diff
@@ -78,7 +82,10 @@ class PayeControllerHistoric @Inject()(
     }
   } recoverWith hodStatusRedirect
 
-  def hodStatusRedirect(
+  private def isRtiAvailable(employments: Seq[Employment]): Boolean =
```

Comment:
Could you rename this to something along the lines of "isRtiUnavailable"?

Would make more sense